test/lit/wasm-split/profile-guided.wast

```diff
@@ -0,0 +1,67 @@
+;; Instrument the binary
+
+;; RUN: wasm-split --instrument %s -o %t.instrumented.wasm
+
+;; Create profiles
+
+;; RUN: node %S/call_exports.mjs %t.instrumented.wasm %t.foo.prof foo
+;; RUN: node %S/call_exports.mjs %t.instrumented.wasm %t.bar.prof bar
+;; RUN: node %S/call_exports.mjs %t.instrumented.wasm %t.both.prof foo bar
+;; RUN: node %S/call_exports.mjs %t.instrumented.wasm %t.none.prof
+
+;; Create profile-guided splits
+
+;; RUN: wasm-split %s --profile=%t.foo.prof -v -o1 %t.foo.1.wasm -o2 %t.foo.2.wasm \
+;; RUN:   | filecheck %s --check-prefix FOO
+
+;; FOO: Keeping functions: deep_foo_callee, foo, foo_callee, shared_callee
```

Comment:
what is `FOO` here and above? I'm not sure how to read this...

Reply:
`FOO` takes the place of `CHECK` (via the --check-prefix option above). This option allows multiple different test outputs to be checked in the same file with each using a different check prefix.

Reply:
Reference: https://llvm.org/docs/CommandGuide/FileCheck.html#the-filecheck-check-prefix-option

Reply:
Thanks! I see, so the relevant docs are for FileCheck and not lit... TIL

I understand the benefit of the prefix in the example there, but not here. It seems like FOO, BAR, are each used once and used immediately. Could they all have been CHECK?

Reply:
If the had all been CHECK, then every invocation of filecheck would have checked all of them against the input, which wouldn't have worked. Remember that filecheck matches against the entire file, not just the lines after or between RUN lines. filecheck doesn't know anything about RUN lines at all.

Reply:
Thanks, I had assumed it did understand about RUN lines from the examples, but now I get it..